tests/components/camera/test_local_file.py

```diff
@@ -59,7 +59,7 @@ def test_file_not_readable(self):
             fp.flush()
 
             with mock.patch('os.access', return_value=False):
-                assert setup_component(self.hass, 'camera', {
+                assert not setup_component(self.hass, 'camera', {
```

Comment:
Oh I see it now. We are able to fail during validation instead of during setup. 👍  bueno